otter/test/convergence/test_gathering.py

```diff
@@ -419,145 +424,128 @@ def test_invalid_first_entry(self):
         self.assertRaises(ValueError, extract_CLB_drained_at, feed)
 
 
+def lb_req(method, url, json_response, response):
+    """
+    Return a SequenceDispatcher two-tuple to match a service request and
+    specify the result.
+    """
+    if isinstance(response, Exception):
+        def handler(i): raise response
+    else:
+        def handler(i): return (StubResponse(200, {}), response)
+    return (
+        Retry(
+            effect=mock.ANY,
+            should_retry=ShouldDelayAndRetry(
+                can_retry=retry_times(5),
+                next_interval=exponential_backoff_interval(2))
+        ),
+        nested_sequence([
+            (service_request(
+                ServiceType.CLOUD_LOAD_BALANCERS,
+                method, url, json_response=json_response).intent,
+             handler)
+        ])
+    )
+
+
+def nodes_req(lb_id, nodes):
+    return lb_req('GET', 'loadbalancers/{}/nodes'.format(lb_id),
+                  True, {'nodes': nodes})
+
+
+def feed_req(lb_id, node_id, response):
```

Comment:
Non-blocking - since we also deal with cloud feeds, and may in the observer in the future, for clarity can we name this `node_feed_req` instead?